Makefile

```diff
@@ -24,7 +24,7 @@ build:
 		-f Dockerfile --target=binary --output=. .
 
 .PHONY: multi-arch-build
-PLATFORMS = linux/amd64,linux/arm64,linux/arm/v7
```

Comment:
I don't see any interest in building ppc64le.

Reply:
Dropped.